IntercomDotnet/Client.cs

```diff
@@ -15,9 +15,9 @@ public class Client
         public string UserName { get; set; }
         public string Password { get; set; }
 
-        public double GetUnixTimestamp(DateTime inputTime)
+        public long GetUnixTimestamp(DateTime inputTime)
         {
-            return (inputTime - new DateTime(1970, 1, 1).ToUniversalTime()).TotalSeconds;
+            return (long)inputTime.ToUniversalTime().Subtract(new DateTime(1970, 1, 1, 0, 0, 0)).TotalSeconds;
```

Comment:
Brilliant, thanks for the research - really appreciated.

Excellent, that does make sense - I just checked the [MSDN docs](https://msdn.microsoft.com/en-us/library/xcfzdy4x%28v=vs.110%29.aspx) and it initialises `DateTime` via that constructor to DateTimeKind.Unspecified. When `ToUniversalTime()` is called on a `DateTime` object of `DateTimeKind.Unspecified`, it appears it will then take the Unspecified to be your local time, and then convert from there to UTC. See explaination on StackOverflow [here](http://stackoverflow.com/questions/1201378/how-does-datetime-touniversaltime-work)

Thanks for the help on this one!